agi.go

```diff
@@ -311,7 +315,8 @@ func (a *AGI) Get(key string) (string, error) {
 
 // GetData plays a file and receives DTMF, returning the received digits
 func (a *AGI) GetData(name string, timeout int, maxdigits int) (digit string, err error) {
-	return a.Command("GET DATA", name, strconv.Itoa(timeout), strconv.Itoa(maxdigits)).Val()
+	resp := a.Command("GET DATA", name, strconv.Itoa(timeout), strconv.Itoa(maxdigits))
+	return resp.ResultString, resp.Error
```

Comment:
It seems this pattern is used a few times (another example is on line 452, in `WaitForDigit`).  It would be nice to add, for instance, the `Result()` method on `Response` and use it, instead.

```go
// Result returns the ResultString of a Response, as well as any error encountered.  Depending on the command, this is sometimes more useful than Val()
func (r *Response) Result() (string,error) {
  return r.ResultString, r.Error
}
```